infrastructure/template.yaml

```diff
@@ -555,6 +555,165 @@ Resources:
         - !Ref AlarmNotificationTopic
       ActionsEnabled: true
 
+  ######################
+  # Deleting the Record
+  ######################
+
+  UserAccountDeletionTopic:
+    Type: AWS::SNS::Topic
+    Properties:
+      DisplayName: UserAccountDeletionTopic
+      KmsMasterKeyId: !Ref SnsKmsKey
+      TopicName: UserAccountDeletion
+
+  DeteUserServicesSubscription:
```

Comment:
```suggestion
  DeleteUserServicesSubscription:
```

Reply:
yeah just spotted that too, forced pushed! thanks!